### PR TITLE
Test Dynamic CSS classes using mock window

### PR DESCRIPTION
**Pull request by jridgewell**
Should resolve the test flake.

Unfortunately, this prevents us from testing the render-delaying aspect. :frowning:

Fixes https://github.com/ampproject/amphtml/issues/2308.

### CODE REVIEW

File: extensions/amp-dynamic-css-classes/0.1/amp-dynamic-css-classes.js [hunk 1]
```diff
@@ -155,7 +156,7 @@ function addRuntimeClasses(win) {
  * @param {!Window} win
  * @return {!Object} All services need to return an object to "load".
  */
```
Comment (cramforce) [inline extensions/amp-dynamic-css-classes/0.1/amp-dynamic-css-classes.js]: `@visiblefortesting`?

**Comment by cramforce** [comment]
LGTM modulo whatever failed. I think you may want to consider to add back the old test as an integration test later on. But only run it once and then you can increase the timeouts a bit.

**Comment by rudygalfi** [comment]
Any update on this? Assuming #2308 is still unresolved?

**Comment by jridgewell** [comment]
Ping @erwinmombay.

File: extensions/amp-brightcove/0.1/test/test-amp-brightcove.js [hunk 2]
```diff
@@ -86,7 +86,7 @@ describe('amp-brightcove', () => {
   });
 
   // TODO(erwinm) unskip this when we figure out why it fails on travis
```
Comment (erwinmombay) [inline extensions/amp-brightcove/0.1/test/test-amp-brightcove.js]: remove todo

**Comment by erwinmombay** [comment]
@jridgewell LGTM.

File: extensions/amp-brightcove/0.1/test/test-amp-brightcove.js (hunk 2 above)
Comment (jridgewell) [inline extensions/amp-brightcove/0.1/test/test-amp-brightcove.js]: Removed.

**Pull request merged**